src/System.Net.Mail/src/System/Net/Mail/SmtpClient.cs

```diff
@@ -748,6 +749,18 @@ public void SendAsync(MailMessage message, object userToken)
             }
         }
 
+        private bool IsSNCInCredentialCache(CredentialCache cache)
+        {
+            // Check if SystemNetworkCredential is in given cache.
+            foreach (var credential in cache)
+            {
+                if (ReferenceEquals(credential, s_defaultNetworkCredential))
```

Comment:
Nit: Please add braces around the `if` statement to follow the style of this file.